news/_posts/core-weekly/2022-07-04-coreweekly-26-2022.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 26 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-07-04 15:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 27th of June to Sunday 3rd of July 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+Multiple new applications have been submitted to join the team: [marsaldev](https://github.com/PrestaShop/open-source/issues/105) for committer position and [PrestaEdit](https://github.com/PrestaShop/open-source/issues/106) for maintainer position. This is great news!
+
+In the meantime, the QA team is still going through the 8.0.0 alpha functional campaign which aims to evaluate how much work would be needed before being able to deliver an 8.0.0 alpha build.
+
+Also the next session of the public PrestaShop demo from the maintainers team will happen [this week, on Wednesday, June 6, 2022 at 4pm CEST](https://build.prestashop.com/news/upcoming-demo-5-2022/)!
```

Comment:
Also the next session of the public PrestaShop demo from the maintainers team will happen [this week, on Wednesday, June 6, 2022 at 4pm CEST](https://build.prestashop.com/news/upcoming-demo-5-2022/)!
```suggestion
Also the next session of the public PrestaShop demo from the maintainers team will happen [this week, on Wednesday, July 6, 2022 at 4pm CEST](https://build.prestashop.com/news/upcoming-demo-5-2022/)!
```